src/nav/telemetry-data-platform.yml

```diff
@@ -145,12 +145,12 @@ pages:
                 path: /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/app-data-nrql-query-examples
               - title: Browser NRQL examples
                 path: /docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/browserspa-nrql-query-examples
-  - title: Manage data
+  - title: Data management
     path: /docs/data-apis/manage-data
     pages:
-      - title: Manage your data
+      - title: The data management hub
```

Comment:
Same question as before. I am not familiar with a "data management hub" but this isn't my area, so feel free to ignore!